moped-editor/src/components/DataTable/DataTable.js

```diff
@@ -388,7 +391,7 @@ const DataTable = ({ fieldConfiguration, data, loading, error, refetch }) => {
       ? lookupTablesData[lookupTable] ?? []
       : fieldConfig?.options ?? [];
 
-    const currentValue = editValue ?? initialValue;
+    const currentValue = editValue ?? (initialValue || "");
```

Comment:
the only `select` field in this component is for "Current Status". It's not possible to set it to empty, but projects can be created w/o an initial "Current Status". This may have been a regression that snuck in at some point, but in any case we will tackle it with [#7036](https://github.com/cityofaustin/atd-data-tech/issues/7036)